src/codeu/chat/server/Model.java

```diff
@@ -71,6 +72,28 @@ public int compare(Time a, Time b) {
   public void add(User user) {
     currentUserGeneration = userGenerations.make();
 
+    /// SQLite
+    Connection c = null; /// Create connection
+    Statement stmt = null;
+    try {
+      Class.forName("org.sqlite.JDBC");
+      c = DriverManager.getConnection("jdbc:sqlite:test.db"); /// Connect to database
```

Comment:
I would make this a constant since it is used in multiple places. Maybe you should consider creating a SQLUtils.java class that contains helper methods and constants that can be reused. You could consider making a helper function for executeSQL(string) which takes int he sql statement that you want to execute.